Source/MoreDotNet.Test/Extensions/Common/RandomExtensions/NextStringTests.cs

```diff
@@ -0,0 +1,148 @@
+﻿namespace MoreDotNet.Tests.Extensions.Common.RandomExtensions
+{
+    using System;
+
+    using MoreDotNet.Extensions.Common;
+
+    using Xunit;
+
+    public class NextStringTests
+    {
+        private const int Zero = 0;
+        private const int OneHundred = 100;
+
+        [Fact]
+        public void NextString_ShouldReturnStringWithLenghtZero()
+        {
+            var random = new Random();
+            var result = random.NextString(Zero);
+
+            Assert.IsType<string>(result);
+            Assert.Equal(Zero, result.Length);
+        }
+
+        [Fact]
+        public void NextString_ShouldReturnString_WithLenght100Chars()
+        {
+            var random = new Random();
+            var result = random.NextString(OneHundred);
+
+            Assert.IsType<string>(result);
+            Assert.Equal(OneHundred, result.Length);
+        }
+
+        [Fact]
+        public void NextString_ShouldReturnString_OfAlphabeticChars()
+        {
+            var random = new Random();
+            var result = random.NextString(OneHundred, MoreDotNet.Models.CharType.AlphabeticAny);
+
+            Assert.IsType<string>(result);
+            Assert.Equal(OneHundred, result.Length);
+            for (int i = 0; i < OneHundred; i++)
```

Comment:
You can do the whole string string Assert in one go:
`                var check = result.Select(x => x.IsLetter()).All(x => x);
                Assert.True(check);`